src/bridger/Bridger.sol

```diff
@@ -201,34 +203,24 @@ contract Bridger is
     }
 
     /**
-     * @dev Bridges deposits in bulk every hour to the L2
+     * @dev Bridges deposits to Kinto
      */
-    function bridgeDeposits(address asset, uint256 maxGas, uint256 gasPriceBid, uint256 maxSubmissionCost)
-        external
-        payable
-        override
-        onlyPrivileged
-    {
+    function bridgeDeposits(address asset, BridgeData calldata bridgeData) external payable onlyPrivileged {
```

Comment:
should bridgeData include amount so we can test first with 1 unit?

Reply:
Yep, lets do it, just to be extra safe!